src/frontend.rs

```diff
@@ -150,6 +155,12 @@ impl SignedPod {
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct AnchoredKey(pub Origin, pub String);
 
+impl Into<middleware::AnchoredKey> for AnchoredKey {
```

Comment:
It's preferable to implement the `From` trait so that we get both `From` and `Into`.
Rust allows implementing a `From` trait on a foreign type without issues.